app/controllers/api/users.php

```diff
@@ -1189,15 +1189,16 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         // clone user object to send to workers
         $clone = clone $user;
 
-        $dbForProject->deleteDocument('users', $userId);
-
-        $queueForDeletes
-            ->setType(DELETE_TYPE_DOCUMENT)
-            ->setDocument($clone);
+        $affected = $dbForProject->deleteDocument('users', $userId);
```

Comment:
Lets move this change to a separate PR, where we can apply this change to all the endpoints that require it